crates/http-sink/src/sink.rs

```diff
@@ -74,6 +76,8 @@ mod test {
             user_agent: "fluvio/http-sink 0.1.0".into(),
             method: "POST".into(),
             headers: vec!["Content-Type: text/html".into()],
+            http_connect_timeout_millis: Duration::from_secs(1),
```

Comment:
we should change this name from `millis` to `secs`. I don't see a use case where we need to detect timeouts in miliseconds.

Reply:
actually I should remove the _millis and _secs I think because we are using `humantime_serde`, I think it handles the units

Reply:
Good catch!

Reply:
BTW Milliseconds is common timeout unit https://www.geeksforgeeks.org/node-js-http-server-timeout-property/amp/.

Reply:
yeah I think both would be ok

Reply:
that should be fine... Duration::from_millis(...) still works.

Reply:
I just removed the units since it handles Duration directly